packages/models-library/src/models_library/api_schemas_webserver/groups.py

```diff
@@ -45,11 +62,41 @@ class GroupGet(OutputSchema):
         default=None, description="url to the group thumbnail"
     )
     access_rights: GroupAccessRights = Field(..., alias="accessRights")
-    inclusion_rules: dict[str, str] = Field(
-        default_factory=dict,
-        description="Maps user's column and regular expression",
-        alias="inclusionRules",
-    )
+
+    inclusion_rules: Annotated[
+        dict[str, str],
+        Field(
+            default_factory=dict,
+            description="Maps user's column and regular expression",
+            alias="inclusionRules",
+        ),
+    ] = DEFAULT_FACTORY
+
+    @classmethod
+    def from_model(cls, group: Group, access_rights: AccessRightsDict) -> Self:
+        # Merges both service models into this schema
+        return cls.model_validate(
+            {
+                **_rename_keys(
+                    group.model_dump(
+                        include={
+                            "gid",
+                            "name",
+                            "description",
+                            "thumbnail",
+                            "inclusion_rules",
+                        },
+                        exclude={"access_rights"},
+                        exclude_unset=True,
+                        by_alias=False,
+                    ),
+                    name_map={
+                        "name": "label",
+                    },
+                ),
+                "access_rights": access_rights,
+            }
+        )
 
     model_config = ConfigDict(
```

Comment:
Since it's an output schema you can do something like this (`validation_alias` exists as well for input ones) and avoid to put the alias in the fields:

```suggestion
    model_config = ConfigDict(
        alias_generator=AliasGenerator(
            serialization_alias=to_camel,
        ),
```

Reply:
that is already in the base class: OutputSchema